### PR TITLE
[IMP] deploy: reword multi-thread/process section

### PR DESCRIPTION
**Pull request by Julien00859**
Many users don't start odoo-bin in --workers (multi-processing) mode but instead leave the default configuration which use the development/demo multi-threaded server, even in production.

This commit rewords the section about the difference between the built-in servers and highlight the many advantages of the multi-processing on. It repeats that information when it comes to running a dedicated cron server, that it should use the multi-processing server instead of the default multi-threading one.

Fixes: odoo/odoo#88984
Closes: odoo/odoo#128571

### CODE REVIEW

**Review by xpl-odoo — COMMENTED** [review]
Hi @Julien00859 
If you are waiting for a review, I can review from a rst guidelines / English grammar perspective.
I won't be able to check the actual technical content, but I assume it's not needed ;)

File: content/administration/install/deploy.rst
Comment (xpl-odoo) [inline content/administration/install/deploy.rst:182]:
```suggestion
Odoo includes built-in HTTP, cron, and live-chat servers, using either
```

File: content/administration/install/deploy.rst
Comment (xpl-odoo) [inline content/administration/install/deploy.rst:188]:
```suggestion
The **multi-threaded** server is a simpler server primarily used for
development, demonstrations, and its compatibility with various operating
systems (including Windows). A new thread is spawned for every new HTTP
request, even for long-running requests such as long polling. Extra daemonic
```

File: content/administration/install/deploy.rst
Comment (xpl-odoo) [inline content/administration/install/deploy.rst:194]:
```suggestion
The multi-threaded server is the default server. For docker containers, it
is selected by leaving the :option:`--workers <odoo-bin --workers>` option out
or setting it to ``0``.
```

File: content/administration/install/deploy.rst
Comment (xpl-odoo) [inline content/administration/install/deploy.rst:205]:
```suggestion
The multi-processing server is opt-in. It is selected by setting
```

File: content/administration/install/deploy.rst
Comment (xpl-odoo) [inline content/administration/install/deploy.rst:369]:
```suggestion
On Linux-like systems, using the multi-processing server over
the multi-threading one is recommended to benefit from better hardware usage and increased
stability, i.e., using the :option:`--workers=-1 <odoo-bin --workers>`
```

File: content/administration/install/deploy.rst
Comment (xpl-odoo) [inline content/administration/install/deploy.rst:382]:
```suggestion
Reusing the Odoo cron server to serve the live chat requests is possible. 
```

**Comment by Julien00859** [comment]
Thanks you :) I'm on holiday this week, I'll do the followup on monday, cheers

File: content/administration/install/deploy.rst
Comment (Julien00859) [reply content/administration/install/deploy.rst]: > The multi-threaded server is the default server, also for docker containers. It
is selected by leaving the :option:`--workers <odoo-bin --workers>` option out
or setting it to ``0``.

File: content/administration/install/deploy.rst
Comment (rco-odoo) [inline content/administration/install/deploy.rst:232]: That's a veeery long line :wink:

File: content/administration/install/deploy.rst
Comment (d-fence) [inline content/administration/install/deploy.rst:382]: I'm not sure that `to reuse` is the appropriate verb to use here.
As it means `to use again especially in a different way`.
But here the cron server is still used as a cron server **and** as a longpolling one.
Why not something like
```suggestion
The Odoo cron server can also be used to serve the live chat requests.
```

**Review by d-fence — COMMENTED** [review]
LGTM with a little comment.

**Review by rco-odoo — APPROVED** [review]
This is way much better.  Thanks!

File: content/administration/install/deploy.rst
Comment (Julien00859) [reply content/administration/install/deploy.rst]: suggestion applied

File: content/administration/install/deploy.rst
Comment (Julien00859) [reply content/administration/install/deploy.rst]: fixed, I also made sure that all other lines where wrap at 100 chars (documentation guidelines)

**Review by d-fence — APPROVED** [review]
Nice work. That was needed, it's much much clearer

**Comment by Julien00859** [comment]
Hello @odoo/documents-doc-review we are ready :)

**Comment by jcs-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
